pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -229,6 +229,17 @@ func (b *BridgePodInterface) preparePodNetworkInterfaces() error {
 		return err
 	}
 
+	// turn TX offload checksum because it causes dhcp failures
+	if err := EthtoolTXOff(b.podInterfaceName); err != nil {
```

Comment:
This need to be the bridge interface ```b.bridgeInterfaceName``` not the pod interface. the dhcp is running on the bridge interface.

Reply:
@SchSeba I think I'm running into some testing problems, I originally had the interface as k6t-eth0 but I get a no such device error on the tests.